src/main/java/com/exasol/releasedroid/adapter/jira/JiraReleaseMaker.java

```diff
@@ -19,6 +19,7 @@
  */
 public class JiraReleaseMaker implements ReleaseMaker {
     private static final Logger LOGGER = Logger.getLogger(JiraReleaseMaker.class.getName());
+    private static final String JIRA_CONFIG_PATH = "release_config.yml";
```

Comment:
Isn't the same file also used for detecting the platforms?
If yes, maybe it would make sense to parse it only once? Or to share some code between the parsers.

Reply:
This is a matter of isolation. If we delete the Jira module in the future, we don't want to affect any code outside of the Jira directory. So the only common logic here - opening the file.

Reply:
yes, + reading it as yml.